src/Uno.UWP/ApplicationModel/Contacts/Contact.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Windows.ApplicationModel.Contacts
+{
+	public partial class Contact
+	{
+		// public string Name { get; set } // can be unimplemented, see firstname (and lastname?)
+		// https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.contacts.contact.name
+
+		public IList<ContactEmail> Emails { get; internal set; }
+		// Address (321 char), Description (512 char), Kind
+		// KIND: Personal, Work, Other
+
+		public IList<ContactAddress> Addresses { get; internal set; }
+		// Country (1024), Kind, PostalCode (zip, 1024), Region (state, 1024), StreetAddress (street, 1024), Description (512), Locality (city, 1024)
+		// KIND: Home, Work, Other
+
+		public IList<ContactPhone> Phones { get; internal set; }
+		// Description (512), Kind, Number (50)
+		// KIND: 10 roznych, Home, Mobile, Work, Other, Pager, BusinessFax, HomeFax, Company, Assistant, Radio
+
+		// public string FullName { get; internal set; }

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4900834)

Reply:
Yes, it was left-over from implementing... but also with some info about UWP limits of len (e.g. Phones.Number has limit of 50 chars), currently this limits are not enforced in Uno implementation.
Remove all this?